kv/db.go

```diff
@@ -146,3 +147,78 @@ func (s *DBServer) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	w.Header().Set("Content-Type", contentType)
 	w.Write(body)
 }
+
+// RegisterRPC ...
+func (s *DBServer) RegisterRPC(rpcServer *rpc.Server) error {
+	return rpcServer.RegisterName("Server", &s.rpcDBServer)
```

Comment:
The "service" namespace is a little confusing: here we register the suite of RPCs under the "Server" service, and in node.go we register the same methods under "Node".